newrelic.yml

```diff
@@ -0,0 +1,65 @@
+#
+# This file configures the New Relic Agent.  New Relic monitors Ruby, Java,
+# .NET, PHP, Python, Node, and Go applications with deep visibility and low
+# overhead.  For more information, visit www.newrelic.com.
+#
+# Generated October 28, 2022
+#
+# This configuration file is custom generated for NewRelic Administration
+#
+# For full documentation of agent configuration options, please refer to
+# https://docs.newrelic.com/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration
+
+common: &default_settings
+  # Required license key associated with your New Relic account.
+  license_key: ${NR_LICENSE_KEY}
+
+  # Your application name. Renaming here affects where data displays in New
+  # Relic.  For more details, see https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/renaming-applications
+  app_name: 'MiniTwit'
+
+  distributed_tracing:
+    enabled: true
+
+  # To disable the agent regardless of other settings, uncomment the following:
+
+  # agent_enabled: false
+
+  # Logging level for log/newrelic_agent.log
+  log_level: info
```

Comment:
Where is `log/newrelic_agent.log` located?
If it is generated on the host machine, then I suggest we add this to the `.gitignore` file.
Might be that the below `development` environment settings makes sure logs are not generated locally, though?

Reply:
It points to the log folder in the root of the repository, which is already git ignored 👍
It will be generated by any machine which runs the application in production mode, which is of course generally the remote host.